# Generated by Django 5.0.1 on 2024-01-27 16:55

from django.db import migrations, models


class Migration(migrations.Migration):

    dependencies = [
        ('company', '0001_initial'),
    ]

    operations = [
        migrations.RemoveField(
            model_name='company',
            name='whatsapp_numbers',
        ),
        migrations.AddField(
            model_name='company',
            name='whatsapp_number',
            field=models.CharField(default=0, max_length=10),
            preserve_default=False,
        ),
        migrations.AddField(
            model_name='whatsappnumber',
            name='companies',
            field=models.ManyToManyField(related_name='whatsapp_numbers', to='company.company'),
        ),
        migrations.AlterField(
            model_name='company',
            name='pan_number',
            field=models.CharField(error_messages={'unique': 'This PAN is already in use.'}, max_length=10, unique=True),
        ),
    ]
